server/controller/controller.js

```diff
@@ -353,7 +300,7 @@ class Controller {
     getSummary(slackId, callback) {
         let reply = '';
         // get players/results
-        DAO.getInstance().getResults(null, null, null, function(resultArray) {
+        fetch('http://localhost:53167/api/results').then(function(resultArray) {
```

Comment:
variables from the fetch aren't consistently named.

Also should really refactor to use arrow notation